service/internal/cloudconfig/template/aws-cni.go

```diff
@@ -177,13 +177,13 @@ spec:
             - name: ADDITIONAL_ENI_TAGS
               value: "{}"
             - name: AWS_VPC_K8S_CNI_LOGLEVEL
-              value: DEBUG
+              value: INFO
             - name: AWS_VPC_K8S_PLUGIN_LOG_LEVEL
-              value: DEBUG
+              value: INFO
             - name: AWS_VPC_K8S_CNI_LOG_FILE
-              value: /host/var/log/aws-routed-eni/ipamd.log
+              value: stdout
             - name: AWS_VPC_K8S_PLUGIN_LOG_FILE
-              value: /var/log/aws-routed-eni/plugin.log
+              value: /host/var/log/aws-routed-eni/plugin.log
```

Comment:
typo that caused the log end sin container and not on the host

Reply:
from aws-cni entrypoint.sh
```bash
validate_env_var()
{
    log_in_json info "Validating env variables ..."
    if [[ "${AWS_VPC_K8S_PLUGIN_LOG_FILE,,}" == "stdout" ]]; then
        log_in_json error "AWS_VPC_K8S_PLUGIN_LOG_FILE cannot be set to stdout"
        exit 1     
    fi 
}

```

Reply:
For my curiosity, does `/dev/stdout` not work? If not, why?

Reply:
No idea, there is nothing written about this in the docs, somehow this log is not even written so I think we don't even use it, maybe @paurosello  knows more?

Reply:
I think this was written and had information in it, i will double check. And maybe try out the /dev/stdout that Tim mentions

Reply:
I tried both,  log file and and `/dev/stdout` but  it does not look like any output is printed

Reply:
I checked the container and there is nothing written as well. The file is not even created so I don't think its used at all.

If we look back at our configuration than this file was never even on the machine, as the path was without the `host` so it would be pointing to the container folder only.

I feel we are safe to leave it like that as we never used it.